Packs/AzureSentinel/Scripts/MicrosoftSentinelSetOwnerButton/MicrosoftSentinelSetOwnerButton.yml

```diff
@@ -0,0 +1,30 @@
+args:
+- name: owner_email
+  required: true
+  description: The owner email to set as user principal name.
```

Comment:
explain how this can be found in the sentinel ui

Reply:
Done